packages/core/src/source/Source.ts

```diff
@@ -313,24 +313,26 @@ export class Source extends ReadonlySource {
 		this.readRemaining()
 		return this
 	}
+}
 
-	static isDigit(c: string): c is Digit {
+export namespace Source {
+	export function isDigit(c: string): c is Digit {
```

Comment:
What's the reason for switching to a namespace instead of static methods?

Reply:
i thought namespaces more "cleanly" specified the intention behind these static functions. i share your question you left here: https://github.com/SpyglassMC/Spyglass/pull/1129#discussion_r1597703484

i think namespaces look cleaner than static functions but am not super committed either way. it'd be nice if we chose one and stuck with it though

i don't think theres a functional difference?